tools/loader/ImageClassifier.cpp

```diff
@@ -77,6 +77,10 @@ llvm::cl::opt<bool> convertInAndOutToFp16(
     llvm::cl::desc(
         "Convert the input and output tensors of the network to fp16"),
     llvm::cl::cat(imageLoaderCat));
+llvm::cl::opt<unsigned> expectedCategoryIndex(
```

Comment:
This allows for only a single index to be specified, even though for the Caffe2 models we allow multiple inputs. Perhaps this could instead be a comma separated list of expected categories, in the same order as the input images order?

Reply:
Fixed in the last commit